javascript/src/main/components/Schedule/AddSchedForm.js

```diff
@@ -4,20 +4,20 @@ import SelectQuarter from 'main/components/BasicCourseSearch/SelectQuarter';
 import { quarterRange } from 'main/utils/quarterUtilities';
 
 const AddSchedForm = ({ createSchedule, updateSchedule, existingSchedule }) => {
+  if (true) {
+    console.log('hi');
+  }
```

Comment:
Hi! This shouldn't be here though.
Also what's the point of doing `if (true)` haha

Reply:
omg this is cute

Reply:
Fixed!

Reply:
Haha that was me trying to figure out how code was executed inside a component. Thanks for the fix Liuhao!